src/plugins/intel_gna/tests/functional/shared_tests_instances/single_layer_tests/conv_low_precision.cpp

```diff
@@ -66,10 +66,8 @@ class ConvLowPrecisionTest : public testing::WithParamInterface<ConvLowPrecision
     }
 
     InferenceEngine::Blob::Ptr GenerateInput(const InferenceEngine::InputInfo& info) const override {
```

Comment:
Are these tests after fixes covering issue fixed in this PR( scale factor calculation with int8 convolution weights) or we should implement new regression tests?

Reply:
The tests are fixed to verify int8 convolution support.